sql/001-create-table-and-load-data.sql

```diff
@@ -1,10 +1,22 @@
 DROP TABLE IF EXISTS names;
 
 CREATE TABLE names (
-  id int unsigned AUTO_INCREMENT,
+  name_id int unsigned AUTO_INCREMENT,
```

Comment:
namesテーブルのidなので
```suggestion
  id int unsigned AUTO_INCREMENT,
```
で良いと思います。